### ISSUE TITLE
feature request: add dark theme option 

### ISSUE THREAD

**Issue by Greylinux** [johnnyq/simpnas#54]
Hi Johnny,

would it be possible to implement a dark theme  for the web interface ?

**Comment by johnnyq** [comment]
@GreyMarvel Good idea! it looks like there is already a bootstrap library for Dark mode here https://bootswatch.com/darkly/ This shouldn't be too difficult to implement, but i'm going to hold off on this until Bootstrap 5 comes out later this year. I'll keep this open.

**Comment by Greylinux** [comment]
This is great 👌 I think every good project should have a dark theme . I look forward to it's progress .

Thanks
